vendor/vendor.cc

```diff
@@ -92,12 +96,13 @@ void RunServer(std::string addr) {
 }
 
 int main(int argc, char** argv) {
-  int server_number = 4;
+  int server_number = 1;
   int base_port = 50053;
+
   std::vector<std::thread> threads;
```

Comment:
This threading is no longer being used, consider removing.

Reply:
Nice catch! Thanks Nick!